intelmq/bots/collectors/mail/collector_mail_body.py

```diff
@@ -29,6 +29,7 @@ def process_message(self, uid, message):
                 report["extra.email_subject"] = message.subject
                 report["extra.email_from"] = ','.join(x['email'] for x in message.sent_from)
                 report["extra.email_message_id"] = message.message_id
+                report["extra.email_received"] = message.date
```

Comment:
Is `message.date` really the *reception* date, not the sending date?

Reply:
I rechecked the `imbox` library you use for emails. It seems it just takes mail header field `Date`, so you should be right

Reply:
Renamed field to the `extra.email_date`